Documents/flashing

```diff
@@ -6,17 +6,21 @@
 
 Visually identify any content that flashes. Flashing is content that rapidly alternates between two or more states that vary significantly in contrast.
 
-When flashing content IS found, the result for Test ID 3.A <span id="OLE_LINK17" class="anchor"></span>(Test Name: 2.3.1-flashing) is **NOT TESTED**.
+-	  It is recommended that testers inquire about occurrences of flashing content. Vendors/developers should make testers aware of the location of any flashing content. (does this belong in this doc? Or e.g. guidance including sampling guide)
+-	  The modified “small safe area” is intended for normal viewing distances or equivalent. If the content is intended for large displays where users may easily decrease the viewing distance, flashing content should be limited to  … (KE: tell tester to change res/screen/browser zoom, size) cs: leave to tool? Using G176 approx, maybe 1/6 of screen?
+-	  Testers with known sensitivities to flashing content should avoid testing flashing content.
+-	  Testers who are viewing their screens at a much closer distance than usual (22 inches for a 15-17” screen) should avoid testing flashing content. (needed?)
 
 #### Check 2.3.1-flashing 
 
 | Test Name      | Test ID | Test Condition                                                                                                                                                 |
 |----------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 2.3.1-flashing | 3.A     | If NO flashing content is found, then this Test Condition DOES NOT APPLY (DNA). If flashing content IS found, then this test should be recorded as NOT TESTED. |
+| 2.3.1-flashing | 3.A     | If NO flashing content is found, then this Test Condition DOES NOT APPLY (DNA). |
 
 **Note**:
 
--   Multiple requirements are specified for conforming flashing content. To determine if requirements are met, a testing tool would be very helpful but is not available at this time. The test process will be updated when a testing tool is identified. Until then, the result “Not Tested” will indicate that flashing content was found.
+-   Multiple requirements are specified for conforming flashing content. To determine if requirements are met, a testing tool would be very helpful but is not available at this time. The test process will be updated when a testing tool is identified. Until then, flashing content that is safely within the “small safe area” is allowed. The area has been reduced in this test process to provide an increased safety zone. .
```

Comment:
More explanation would be helpful: 
[general flash and red flash thresholds](https://www.w3.org/WAI/WCAG21/Understanding/three-flashes-or-below-threshold.html#dfn-general-flash-and-red-flash-thresholds), Note 1:

> NOTE
For general software or Web content, using a 341 x 256 pixel rectangle anywhere on the displayed screen area when the content is viewed at 1024 x 768 pixels will provide a good estimate of a 10 degree visual field for standard screen sizes and viewing distances (e.g., 15-17 inch screen at 22-26 inches). (Higher resolutions displays showing the same rendering of the content yield smaller and safer images so it is lower resolutions that are used to define the thresholds.)

Reply:
This seems to be about 1/6 of the screen at normal viewing distance - is that a reasonable max area to use?

Reply:
how are you getting 1/6?

Reply:
alright, help me with the math here - 

- 1024/3 = 341
- 768/3= 256

I get 1/9.

Unless the fractional value is necessary for testing, I would not include it. You'll need to explain the math. (And, it would be easier and more reliable to copy from the WCAG definition.)

Reply:
ah, that's right. doh!

Reply:
Thoughts on requiring warnings, or not allowing direct access to pages with flashing (outside 1/9 area?), without warnings first or user consent?